kubernetes/EKS/setup_efs.sh

```diff
@@ -25,19 +26,17 @@ aws ec2 authorize-security-group-ingress --group-id $MOUNT_TARGET_GROUP_ID --pro
 
 # Create an EFS file system.
 echo "Creating EFS Fils System"
-FILE_SYSTEM_ID=$(aws efs create-file-system | jq --raw-output '.FileSystemId')
+FILE_SYSTEM_ID=$(aws efs create-file-system --region "$REGION" --performance-mode generalPurpose --query 'FileSystemId' --output text)
 echo "Created EFS - $FILE_SYSTEM_ID"
 
-aws efs describe-file-systems --file-system-id $FILE_SYSTEM_ID
+aws efs describe-file-systems --file-system-id $FILE_SYSTEM_ID --region "$REGION"
 
 echo "Waiting 30s for before procedding"
 sleep 30
 
 # Create Mount target in Subnets
 echo "Obtaining Subnets"
-TAG1=tag:kubernetes.io/cluster/$CLUSTER_NAME
-TAG2=tag:kubernetes.io/role/elb
-subnets=($(aws ec2 describe-subnets --filters "Name=$TAG1,Values=shared" "Name=$TAG2,Values=1" | jq --raw-output '.Subnets[].SubnetId'))
+subnets=($(aws ec2 describe-subnets --region "$REGION" --filters "Name=vpc-id,Values=$VPC_ID" | jq --raw-output '.Subnets[].SubnetId'))
```

Comment:
Where is the $VPC_ID being set?

Reply:
https://github.com/pytorch/serve/blob/effef72498dee99c6bfed926c25794bfa1d90751/kubernetes/EKS/setup_efs.sh#L10